Loop/Managers/DeviceDataManager.swift

```diff
@@ -871,6 +851,36 @@ class DeviceDataManager: CarbStoreDelegate, TransmitterDelegate, ReceiverDelegat
 
     let doseStore: DoseStore
 
+    // MARK: DoseStoreDelegate
+
+    func doseStore(doseStore: DoseStore, hasEventsNeedingUpload pumpEvents: [(objectID: NSManagedObjectID, date: NSDate, dose: DoseEntry?, raw: NSData?)], withCompletion completionHandler: (uploadedObjects: [NSManagedObjectID]) -> Void) {
+
+        guard let uploader = remoteDataManager.nightscoutUploader, device = rileyLinkManager.firstConnectedDevice, pumpModel = pumpState?.pumpModel else {
```

Comment:
Haha so I'm hearing we should include everything? What's the preferred syntax?

```
iCorder-6s:com.loudnate.Loop:Upstairs-RileyLink:123456//
```

For all those identifiers above, only the RileyLink name isn't directly known at the time of the delegate call, but I can change that by adding a device ID string to the DoseStore PumpEvent schema (Pump ID is already there).

I filed https://github.com/loudnate/LoopKit/issues/48 to handle the storage and #98 to handle the identifier format.